### ISSUE TITLE
Material UI issue with ESM build 

### ISSUE THREAD

**Issue by dwitzig** [guidone/lets-form#36]
Hi @guidone, just comming back to this project after a few distractions, great to see so many more versions :)

looks like there is a issue with material ui framework since moving to ESM build @ version 0.7.10

I get the following error for every MUI component 
```
ERROR in ./node_modules/lets-form/dist/react-material-ui-esm/index-BB1rB772.js 3:0-52
Module not found: Error: Can't resolve '@mui/material/FormControl' in '/cra-example/node_modules/lets-form/dist/react-material-ui-esm'
Did you mean 'index.js'?
BREAKING CHANGE: The request '@mui/material/FormControl' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

version  0.7.9 works fine. im using the [Export CRA] function in designer to test

**Comment by guidone** [comment]
Thanks for the feedback, I'll give it a check.
0.7.9 is rather old, have you checked with the latest? (the problem should be present as well, it didn't change much MUI in the latest versions)

**Comment by dwitzig** [comment]
yeah first noticed on the latest version 10.6. 
worked my way backwards to find where it was introduced

**Comment by guidone** [comment]
Uh great I didn't understand you already narrowed down where it first appeared.

The problem appears to be with WebPack, [here](https://stackoverflow.com/questions/70964723/webpack-5-in-ceate-react-app-cant-resolve-not-fully-specified-routes) the quick solution, I will integrate in the Designer soon.

The `0.7.10` only triggered the problem since I started importing only the needed component and not the whole MUI library.

Thanks for the feedback

**Comment by dwitzig** [comment]
thanks @guidone. confirming the webpack override works 👍

**Comment by guidone** [comment]
Fixed in LF Designer `0.9.4`.

Thanks for the feedback 🖖
